packages/esm-patient-registration-app/src/patient-registration/form-manager.test.ts

```diff
@@ -8,7 +8,6 @@ const formValues: FormValues = {
   givenName: '',
   middleName: '',
   familyName: '',
-  unidentifiedPatient: false,
```

Comment:
But this should be the default right? At least for a blank patient.

Reply:
Actually, I'm using the attribute value as the source of truth to determine whether the patient is known or not.
I found this field redundant, as it increased the overhead of keeping the track of this above field and the attribute field